prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -222,10 +222,16 @@ func handle(ghc githubClient, roc repoownersClient, log *logrus.Entry, reviewerC
 				if err != nil {
 					return err
 				}
+				added := 0
```

Comment:
`var added int` since you're not initializing anything

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style